app/src/main/java/org/thoughtcrime/securesms/database/Storage.kt

```diff
@@ -1542,6 +1546,34 @@ open class Storage(
         }
     }
 
+    /**
+     * This will create a control message used to indicate that you have accepted a message request
+     */
+    override fun insertMessageRequestResponseFromYou(threadId: Long){
+        val userPublicKey = getUserPublicKey() ?: return
+
+        val mmsDb = DatabaseComponent.get(context).mmsDatabase()
+        val message = IncomingMediaMessage(
```

Comment:
I still think IncomingMediaMessage should be converted to Kotlin and we then use named parameters - because this constructor is just asking for mistakes.